javagen/src/main/java/com/azure/autorest/mapper/ProxyMethodMapper.java

```diff
@@ -124,6 +125,14 @@ public static ProxyMethodMapper getInstance() {
         }
         builder.responseContentTypes(responseContentTypes);
 
+        // Low-level client only requires one request per operation
+        if (settings.isLowLevelClient()) {
+            List<Request> list = operation.getRequests();
+            if (!list.isEmpty()) {
+                operation.setRequests(Collections.singletonList(list.get(0)));
```

Comment:
You should handle it when processing the `requests`, than override it in `operation`.

Reply:
Fixed